tools/integration_tests/read_cache/helpers_test.go

```diff
@@ -136,6 +136,22 @@ func unmountGCSFuseAndDeleteLogFile() {
 	}
 }
 
+func remountGCSFuseAndValidateCacheDeleted(flags []string, t *testing.T) {
+	setup.SetMntDir(rootDir)
+	unmountGCSFuseAndDeleteLogFile()
+	mountGCSFuse(flags)
+	setup.SetMntDir(mountDir)
+
+	cacheSize, err := operations.DirSize(cacheLocationPath)
```

Comment:
Shouldn't we use this function `validateFileIsNotCached(testFileName,t)` ?

Reply:
Checking that the directory size is 0 covers all the scenarios I feel. Also we will validate that file is not cached as part of the read operation after remount.